samples/Compute/platform-image-policy/azurepolicy.json

```diff
@@ -3,8 +3,21 @@
     "name": "platform-image-policy",
     "properties": {
         "displayName": "Only allow a certain VM platform image",
-        "description": "This policy ensures that only UbuntuServer, Canonical is allowed from the image repository",
-        "parameters": {},
+        "description": "This policy ensures that only certain VM platform images are allowed from the image repository",
+        "parameters": {
+            "imagePublishers": {
+                "type": "array",
+                "defaultValue": "Canonical"
```

Comment:
should be inside an array, i.e. ["Canonical"]

Reply:
(same for all parameters)

Reply:
Actually...default values don't make sense for this policy (sorry, I think I suggested it earlier). Each organization will have very different requirements for which VM images they will allow, and Ubuntu probably isn't what most would expect as a default (especially not from Microsoft 😉). Please remove the defaultValue from each parameter, and instead use these example values (["Canonical"], etc) in the powershell and cli examples.